scripts/update_lti.js

```diff
@@ -106,32 +108,32 @@ var updateToolsForCourse = function(course, index, callback) {
     return callback();
   }
 
-  CanvasAPI.getExternalToolsForCourse(course, function(err, courseTools) {
-    if (err) {
-      log.error({'err': err, 'course': course.id}, 'Failed to get external tool configurations for course');
+  CanvasAPI.getExternalToolsForCourse(course, function(apiErr, courseTools) {
+    if (apiErr) {
+      log.error({'err': apiErr, 'course': course.id}, 'Failed to get external tool configurations for course');
       results.error = results.error.concat(toolsToUpdate);
       return callback();
     }
 
-    async.eachOfSeries(toolsToUpdate, function(tool, index, done) {
+    async.eachOfSeries(toolsToUpdate, function(tool, toolIndex, done) {
```

Comment:
It looks right to me as well, @johncrossman. Only thing I'd suggest is that if we rename here, we also rename `index` to `toolIndex` in the sister function `updateToolsForAccount` below; just for consistency.